javamelody-core/src/main/java/net/bull/javamelody/MonitoringFilter.java

```diff
@@ -198,7 +198,7 @@ private void doFilter(FilterChain chain, HttpServletRequest httpRequest,
 			// on binde la requête http (utilisateur courant et requête complète) pour les derniers logs d'erreurs
 			httpRequest.setAttribute(CounterError.REQUEST_KEY, completeRequestName);
 			CounterError.bindRequest(httpRequest);
-			chain.doFilter(wrappedRequest, wrappedResponse);
+			chain.doFilter(httpRequest, httpResponse);

```

Comment:
You have changed `chain.doFilter(wrappedRequest, wrappedResponse);` to 'chain.doFilter(httpRequest, httpResponse);'.
But initial code is there for a reason. Given your change, there would be a regression in features. So it can't be merged.

Instead of this change, I suggest to find why this issue happens in arcgis server. And for example, file an issue with ESRI support.